gm4_trash_cans/data/gm4_trash_cans/functions/disable/wet.mcfunction

```diff
@@ -0,0 +1,18 @@
+# @s = trash_can
+# run from check_dropper
+
+# Water cools down the magma block
+execute if block ~ ~-1 ~ water run tag @s add gm4_trash_disable
+execute if block ~ ~1 ~ water run tag @s add gm4_trash_disable
+execute if block ~-1 ~ ~ water run tag @s add gm4_trash_disable
+execute if block ~1 ~ ~ water run tag @s add gm4_trash_disable
+execute if block ~ ~ ~-1 water run tag @s add gm4_trash_disable
+execute if block ~ ~ ~1 water run tag @s add gm4_trash_disable
```

Comment:
Could use a predicate here, but idk if it's worth it

Reply:
I haven't used predicates before. For now, I'm going to just do what Adante said in the review above.